apis/datadoghq/v2alpha1/datadogagent_types.go

```diff
@@ -198,16 +223,16 @@ type CWSFeatureConfig struct {
 	// +optional
 	Enabled *bool `json:"enabled,omitempty"`
 
+	// EnableSyscallMonitor enables Syscall Monitoring (recommended for troubleshooting only).
+	// Default: false
+	// +optional
+	EnableSyscallMonitor *bool `json:"enableSyscallMonitor,omitempty"`
```

Comment:
i wonder if we should have passive voice to be parallel with the `Enabled` parameters; i.e. `SyscallMonitorEnabled`